docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md

```diff
@@ -0,0 +1,547 @@
+---
+id: eks
+title: Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service
+keywords:
+ - Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service
+ - Replicated PV Mayastor Installation on EKS
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Elastic Kubernetes Service
+ - EKS
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service
```

Comment:
What is the main difference between this and the above document?

Reply:
Both are for EKS! One for instance store like Local SSD in GKE & the other one is without Local SSD meaning disks can be attached manually to those instances. @avishnu